src/components/LoginPage/LoginPage.jsx

```diff
@@ -14,26 +14,52 @@ const StyledContainer = styled.div`
    display: flex;
    flex-direction: column;
    justify-content: space-between;
+   @media only screen and (min-width: 768px) {
+      justify-content: center;
+      align-items: center;
+   }
 `;
 
 const StyledTitle = styled.h2`
-   display: flex;
-   justify-content: center;
+   display: block;
+   margin-left: auto;
+   margin-right: auto;
    color: #024e6b;
    font-size: 48px;
    font-weight: 700;
+   text-align: center;
+   @media only screen and (min-width: 767px) {
+      margin-left: auto;
+      margin-right: auto;
+      padding: 40px;
+   }
 `;
 const StyledImage = styled.img`
    display: flex;
    justify-content: center;
+   @media only screen and (min-width: 768px) {
+      max-width: 200%;
+      height: auto;
+   }
```

Comment:
Can we make the max-width of this a specific number? It looks a bit off when the screen gets wider around like 700px. I think whatever the size is for larger screens should do.


![Screen Shot 2021-04-18 at 10 18 37 PM](https://user-images.githubusercontent.com/42504462/115185166-29186280-a094-11eb-86e0-7c21e31ce0d7.png)